frontend/src/common/queries/collections.ts

```diff
@@ -153,12 +157,16 @@ function fetchCollection(allCollections: CollectionResponsesMap | undefined) {
 export function useCollection({
   id = "",
   visibility = VISIBILITY_TYPE.PUBLIC,
-}) {
+}: {
+  id?: string;
+  visibility: VISIBILITY_TYPE;
+}): UseQueryResult<Collection | TombstonedCollection | null> {
   const { data: collections } = useCollections();
   const queryFn = fetchCollection(collections);
+
   return useQuery<Collection | TombstonedCollection | null>(
     [USE_COLLECTION, id, visibility, collections],
-    queryFn,
+    () => queryFn(id, visibility),
```

Comment:
https://react-query.tanstack.com/guides/migrating-to-react-query-3#query-key-partspieces-are-no-longer-automatically-spread-to-the-query-function